webapp/client/src/components/FormHeader.js

```diff
@@ -16,7 +16,11 @@ export default function FormHeader({ title, intro, hideIntro }) {
 
 FormHeader.propTypes = {
   title: PropTypes.string.isRequired,
-  intro: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  intro: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.element,
+    PropTypes.object,
```

Comment:
Should we use PropTypes.any?

Reply:
Because it already can be an element and an object? However, it can't be a number.